src/components/nav-tabs/CommunityCloud.js

```diff
@@ -1,33 +0,0 @@
-import { Button, Container, Text, Title, Group, List } from '@mantine/core';
```

Comment:
@bryanmontalvan I moved this content to the `community-cloud.js` page itself. I think this makes hacking easier, if we have all the content in the `pages/` directory and only maintain reusable components in the `components` directory. Like the navbar, footer, etc.

Reply:
That works with me :+1: